packages/celotool/src/cmds/generate/faucet-load-test.ts

```diff
@@ -0,0 +1,95 @@
+/* tslint:disable no-console */
+import { newKit } from '@celo/contractkit'
+import { switchToClusterFromEnv } from 'src/lib/cluster'
+import { convertToContractDecimals } from 'src/lib/contract-utils'
+import { addCeloEnvMiddleware, CeloEnvArgv, envVar, fetchEnv } from 'src/lib/env-utils'
+import { AccountType, generateAddress } from 'src/lib/generate_utils'
+import { portForwardAnd } from 'src/lib/port_forward'
+import yargs from 'yargs'
+
+interface FaucetLoadTest extends CeloEnvArgv {
+  count_from: number
+  gold: number
+  dollars: number
+  count_to: number
+  threads_from: number
+  threads_to: number
+}
+
+export const command = 'faucet-load-test'
+
+export const describe = 'command for fauceting the addresses used for load testing'
+
+export const builder = (argv: yargs.Argv) => {
+  return addCeloEnvMiddleware(
+    argv
+      .option('gold', {
+        type: 'number',
+        description: 'Celo Gold amount to transfer',
+        default: 10,
+      })
+      .option('dollars', {
+        type: 'number',
+        description: 'Celo Gold amount to transfer',
+        default: 10,
+      })
+      .option('count_from', {
+        type: 'number',
+        description: 'Index count from',
+        demandOption: 'Please specify a key index',
+      })
+      .option('count_to', {
+        type: 'number',
+        description: 'Index count to',
+        demandOption: 'Please specify a key index',
+      })
+      .option('threads_from', {
+        type: 'number',
+        description: 'Index of key to generate',
+        demandOption: 'Please specify a key threads_from',
+      })
+      .option('threads_to', {
+        type: 'number',
+        description: 'Index of key to generate',
+        demandOption: 'Please specify a key threads_to',
+      })
+  )
+}
+
+export const handler = async (argv: CeloEnvArgv & FaucetLoadTest) => {
+  await switchToClusterFromEnv(argv.celoEnv)
+  const accountType = AccountType.LOAD_TESTING_ACCOUNT
+  const mnemonic = fetchEnv(envVar.MNEMONIC)
+
+  const cb = async () => {
+    const kit = newKit('http://localhost:8545')
+    const account = (await kit.web3.eth.getAccounts())[0]
+    console.log(`Using account: ${account}`)
+    kit.defaultAccount = account
+
+    const [goldToken] = await Promise.all([kit.contracts.getGoldToken()])
+    const [stableToken] = await Promise.all([kit.contracts.getStableToken()])
```

Comment:
Why use arrays and `Promise.all` if the two lines are done in sequence?  Since the two are different types, I would lean towards just simplifying this by not using arrays and `Promise.all`

Reply:
Done thanks for the advice.